azure-pipelines.yml

```diff
@@ -52,9 +52,11 @@ extends:
                 displayName: Use Node 16.x
                 inputs:
                   versionSpec: 16.x
-            
+              # PUPPETEER_DOWNLOAD_BASE_URL=https://storage.googleapis.com/chrome-for-testing-public is a workaround for puppeteer 
+              # as puppeteer versions <22.0.0 will cause npm install to hang in CI environments.
```

Comment:
Real fix is bumping puppeteer to v22 and up but that's a bigger chunk of work as tests fail and node would need to be bumped to 18